### ISSUE TITLE
Some questions about the output data

### ISSUE THREAD

**Issue by Jeff37** [macaodha/batdetect2#17]
Guys,
I like a lot using batDetect2, it's open, fast and efficient :-). So I'm developing a shiny app (under R, I plan to upload it on my GitHub soon) that is calling batDetect2 through a convenient GUI. I'm a bit stuck when it come to understand some numbers from the output.
1. "probability present" (console)
I cannot understand how this value is calculated? I've been checking all the numbers from json files and csv table, as well as from the --spec_features table output. I also read -thoroughly enough I think- your paper "Towards a General Approach for Bat Echolocation Detection and Classification" without finding answer. Could you tell me how it is calculated or where I can find the info? In the following chunk of the output, the probability of 0.676 is retunrned for Nnoc, 0.297 for Nleii and 0.026 for Eser but I cannot relate these probabilities to the numbers in the csv files.
[PaRecAR863806_20230609_214859.wav_spec_features.csv](https://github.com/macaodha/batdetect2/files/12279909/PaRecAR863806_20230609_214859.wav_spec_features.csv)
[PaRecAR863806_20230609_214859.wav.csv](https://github.com/macaodha/batdetect2/files/12279910/PaRecAR863806_20230609_214859.wav.csv)

```
...
4 /home/jf/tmp/TESTfakeAudiomothFolder/20010911_WTC/A/PaRecAR863806_20230609_214859.wav
10 call(s) detected above the threshold.
species name                  probablity present
Nyctalus noctula              0.676
Nyctalus leisleri             0.297
Eptesicus serotinus           0.026
...
```
Could you tell me how I can recalculate these numbers by myself or where to find this info?

2. I'm just wondering what "_bb" stands for in the spec feature csv table? For instance, what is the difference between _max_power_bb_ and _max_power_ ?

Hope you can help me with these points.
Best,
JF Godeau

**Comment by macaodha** [comment]
Regarding your second question.

`max_power_bb' is briefly described here:  
(https://github.com/macaodha/batdetect2/blob/main/batdetect2/detector/compute_features.py#L82)[https://github.com/macaodha/batdetect2/blob/main/batdetect2/detector/compute_features.py#L82]

and `max_power' is briefly described here:  
(https://github.com/macaodha/batdetect2/blob/main/batdetect2/detector/compute_features.py#L127)[https://github.com/macaodha/batdetect2/blob/main/batdetect2/detector/compute_features.py#L127] 

Basically, `max_power' computes the max power across all frequency bands within the time interval of the detect call, whereas `max_power_bb' only computes within the range of the detected lower and upper frequency range.

**Comment by Jeff37** [comment]
Dear @macaodha ,
Thank you for your detailed answer. All clear! After looking into the code you pointed, I understand how much trivial was my question if I would know better python functions. The L240-L254 of _compute_features.py_ are very informative too. I'll try to look better in the code next time before asking questions ;-)
Your answer helped me a lot!

**Comment by macaodha** [comment]
Great - glad your issues were resolved.

**Comment by chrmue44** [comment]
@Jeff37 I am currently developing a software to organize and analyze bat calls (with more or less shiny GUI :-) ) Maybe you are interested : https://github.com/chrmue44/BatInspector

Cheers 
Christian

**Comment by chrmue44** [comment]
@Jeff37 A nice installer is missing yet, I will hopefully do it in a few weeks. You can contact me directly via email chrmue44 [at) gmail (dot] com if you want to.
